Sandbox logs:
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"66541942-ffa5-4fce-b87a-52b84ae701a3\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"66541942-ffa5-4fce-b87a-52b84ae701a3\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 0
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1b13d460-afd5-4dd2-bda5-3e7a6e36b027\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1b13d460-afd5-4dd2-bda5-3e7a6e36b027\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ad91e408-2eab-4a37-935e-3cf5596d1e76\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ad91e408-2eab-4a37-935e-3cf5596d1e76\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"98a7d0ce-e32b-4df5-8f9e-3a279cc439dd\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"98a7d0ce-e32b-4df5-8f9e-3a279cc439dd\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"c7092362-da60-4219-958a-c54c6ea3e6e8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"c7092362-da60-4219-958a-c54c6ea3e6e8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"12b47cd2-0e35-4b15-8633-b76d9086d395\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"12b47cd2-0e35-4b15-8633-b76d9086d395\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"37cec284-9c1d-4db7-89cb-d97c71d0b3f7\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"37cec284-9c1d-4db7-89cb-d97c71d0b3f7\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0b8144f6-4fa9-438d-ae05-6d93e9763d7f\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0b8144f6-4fa9-438d-ae05-6d93e9763d7f\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"51bb6027-985c-43ac-b3d9-9627eaaec003\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"51bb6027-985c-43ac-b3d9-9627eaaec003\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d7cfbe00-4c0f-4296-a182-16d94dcfdf5a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d7cfbe00-4c0f-4296-a182-16d94dcfdf5a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"225bd6cd-ff17-4a2a-93b8-5d1f943f6355\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"225bd6cd-ff17-4a2a-93b8-5d1f943f6355\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"757ea827-5157-42d6-aa6e-357fea70b38f\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"757ea827-5157-42d6-aa6e-357fea70b38f\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"cb114957-ae9b-4929-8bad-ce87542e22b1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"cb114957-ae9b-4929-8bad-ce87542e22b1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ca0b7ab5-383d-429c-8325-c4ec5622f26c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ca0b7ab5-383d-429c-8325-c4ec5622f26c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ac8798ea-6b09-44e8-bc09-f78ff14f2de7\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ac8798ea-6b09-44e8-bc09-f78ff14f2de7\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"38d7e4dc-e689-47a1-8d80-119beb51dbc1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"38d7e4dc-e689-47a1-8d80-119beb51dbc1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"469aad6f-fc6d-428f-8d4e-d73b239f0ad8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"469aad6f-fc6d-428f-8d4e-d73b239f0ad8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2a855040-fa40-4015-83a8-dd4e616927a0\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2a855040-fa40-4015-83a8-dd4e616927a0\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2424f0b7-80be-440e-8880-3d947a671096\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2424f0b7-80be-440e-8880-3d947a671096\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f547d885-1b4e-4789-8036-f86f43c05982\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f547d885-1b4e-4789-8036-f86f43c05982\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 1900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d8f5668f-a776-43be-b067-48bc6378be08\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d8f5668f-a776-43be-b067-48bc6378be08\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"693e3b5c-277b-4caa-b566-0c35c930c923\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"693e3b5c-277b-4caa-b566-0c35c930c923\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"3678b28f-0f5c-471f-a82d-ea67fda35640\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"3678b28f-0f5c-471f-a82d-ea67fda35640\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f3763d81-ceaf-4549-a927-92eb4953103f\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f3763d81-ceaf-4549-a927-92eb4953103f\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"bb0cbfd4-ca9e-4257-bbb8-59b964b0ad6e\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"bb0cbfd4-ca9e-4257-bbb8-59b964b0ad6e\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"c3175d2a-eada-4553-b3a9-842921cba8c1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"c3175d2a-eada-4553-b3a9-842921cba8c1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fc4b4714-efe8-4a2d-870f-adfd0786958e\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fc4b4714-efe8-4a2d-870f-adfd0786958e\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"76c39ccf-ec9a-4176-926c-51ca1fb4141a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"76c39ccf-ec9a-4176-926c-51ca1fb4141a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1987a449-09cb-4401-b2a6-ca8b9ed1b2ec\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1987a449-09cb-4401-b2a6-ca8b9ed1b2ec\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ac15a772-f0b1-4fd4-8239-9b547ee20303\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"ac15a772-f0b1-4fd4-8239-9b547ee20303\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 2900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"55836756-1aa8-4ca4-9c98-62c12d31c451\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"55836756-1aa8-4ca4-9c98-62c12d31c451\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"52a6ce79-3993-4481-9dc1-1206b6ba5662\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"52a6ce79-3993-4481-9dc1-1206b6ba5662\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0747b97d-02a8-47a7-9f43-b7fa4ec53d52\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0747b97d-02a8-47a7-9f43-b7fa4ec53d52\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1d4789ac-0dd2-45cf-afc4-f4bf2643ae65\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1d4789ac-0dd2-45cf-afc4-f4bf2643ae65\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"aaeb91bb-3d44-47cd-bdc9-39c5f37bc995\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"aaeb91bb-3d44-47cd-bdc9-39c5f37bc995\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"cc757068-697f-4d3a-a55e-bd821a312344\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"cc757068-697f-4d3a-a55e-bd821a312344\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2918d713-72e8-4b2b-840d-be7e83f1d1f4\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2918d713-72e8-4b2b-840d-be7e83f1d1f4\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"a22e77d9-f432-4223-81f0-86f5d183e08b\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"a22e77d9-f432-4223-81f0-86f5d183e08b\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b8660eff-4041-47cf-857e-4b80a7e30ca0\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b8660eff-4041-47cf-857e-4b80a7e30ca0\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d2f5a662-fc87-48b0-93af-37ee94c141ef\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d2f5a662-fc87-48b0-93af-37ee94c141ef\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 3900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"6d49534a-944d-4872-8d9d-d71c03ad4938\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"6d49534a-944d-4872-8d9d-d71c03ad4938\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"c656875e-df8c-4b28-b39f-845247f075af\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"c656875e-df8c-4b28-b39f-845247f075af\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"475a5953-277a-4b08-849a-088f20129f0a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"475a5953-277a-4b08-849a-088f20129f0a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"8f553fbc-1433-4501-9430-bf3b3587c9ac\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"8f553fbc-1433-4501-9430-bf3b3587c9ac\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b579c74d-839c-445b-a2a0-a9a0eb1e4748\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b579c74d-839c-445b-a2a0-a9a0eb1e4748\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"222b6beb-2d06-4700-a6e5-c5d790ccab57\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"222b6beb-2d06-4700-a6e5-c5d790ccab57\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"47b02f26-2409-4202-a963-b646b8de5e23\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"47b02f26-2409-4202-a963-b646b8de5e23\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fc1c5ac5-3548-49b5-a252-cc839163d116\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fc1c5ac5-3548-49b5-a252-cc839163d116\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"deaee829-e0fc-4c95-8625-6afd3198fa22\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"deaee829-e0fc-4c95-8625-6afd3198fa22\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"897b11fd-34ce-4ee1-9a9a-5ed31fd56555\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"897b11fd-34ce-4ee1-9a9a-5ed31fd56555\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 4900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"4d064b3d-640e-431c-b7e6-467e88f626dd\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"4d064b3d-640e-431c-b7e6-467e88f626dd\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"4716e563-9066-4921-8c18-b57928711427\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"4716e563-9066-4921-8c18-b57928711427\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"09fcfcd3-f911-4963-98f3-0d0f71b6b6c8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"09fcfcd3-f911-4963-98f3-0d0f71b6b6c8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"940f4dd1-03d5-4fec-bfdd-0c623cf2cbfb\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"940f4dd1-03d5-4fec-bfdd-0c623cf2cbfb\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"16d024cc-f2dd-471b-bc99-05ac7d98adc6\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"16d024cc-f2dd-471b-bc99-05ac7d98adc6\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"e5c28136-0255-4a10-a7f7-7615ce07f1ce\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"e5c28136-0255-4a10-a7f7-7615ce07f1ce\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5a3f712c-9c20-4fad-b7c2-8a6adcf8eabb\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5a3f712c-9c20-4fad-b7c2-8a6adcf8eabb\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5167b796-7e9d-43ad-aa9e-9597af123c90\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5167b796-7e9d-43ad-aa9e-9597af123c90\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f8a321de-7979-4330-bb78-188419168a8c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f8a321de-7979-4330-bb78-188419168a8c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fc174047-da23-40ba-a3d1-ed0e7d995839\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fc174047-da23-40ba-a3d1-ed0e7d995839\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 5900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"45534483-71ba-4bca-9f50-ca19a56ba905\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"45534483-71ba-4bca-9f50-ca19a56ba905\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b15ab159-7187-4a22-b857-5cce285efea8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b15ab159-7187-4a22-b857-5cce285efea8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5c1a08d1-c7ef-4454-837f-396eaa63cc98\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5c1a08d1-c7ef-4454-837f-396eaa63cc98\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"97b7d950-b36b-470f-bb53-083ca2b87bcf\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"97b7d950-b36b-470f-bb53-083ca2b87bcf\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b17a524e-d1e5-4d47-938d-f7d7300711b9\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"b17a524e-d1e5-4d47-938d-f7d7300711b9\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"7245156f-b6f5-4345-afe3-dc50f0d88e50\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"7245156f-b6f5-4345-afe3-dc50f0d88e50\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f8050807-a5bd-4b50-a687-0df3f05f72d4\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f8050807-a5bd-4b50-a687-0df3f05f72d4\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0c460654-c79e-4da0-9422-d301578befd2\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0c460654-c79e-4da0-9422-d301578befd2\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0e08acd1-d01f-4e47-a9f2-a2218f470725\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0e08acd1-d01f-4e47-a9f2-a2218f470725\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2487d9fb-8ab9-422c-9457-36b23b1aefd3\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2487d9fb-8ab9-422c-9457-36b23b1aefd3\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 6900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0a09c1be-e640-4801-aaba-39efcb314ab8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"0a09c1be-e640-4801-aaba-39efcb314ab8\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1575e531-a821-4827-944e-6f7836855cbe\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"1575e531-a821-4827-944e-6f7836855cbe\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"9fb16ac3-9cb2-4396-ac5e-15de87097db5\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"9fb16ac3-9cb2-4396-ac5e-15de87097db5\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"eb0a38e6-d753-423b-83fe-5068be2f5ff4\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"eb0a38e6-d753-423b-83fe-5068be2f5ff4\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f6e2a1bc-2f08-451f-a844-9f633328625b\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f6e2a1bc-2f08-451f-a844-9f633328625b\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"8a91cf29-e14e-442c-aa2f-b9dfa27c547a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"8a91cf29-e14e-442c-aa2f-b9dfa27c547a\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"e255d517-928e-4a41-b8d3-6c746dcb7774\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"e255d517-928e-4a41-b8d3-6c746dcb7774\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2723ec4a-f709-4576-b2dc-e41c20f26e8d\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"2723ec4a-f709-4576-b2dc-e41c20f26e8d\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"59065064-4ef4-47df-a9a3-46fff608c7de\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"59065064-4ef4-47df-a9a3-46fff608c7de\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"db195a26-57f4-4b40-93b9-23f7171975f9\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"db195a26-57f4-4b40-93b9-23f7171975f9\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 7900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d794f994-d939-48de-a909-cf2fc8b5c4b5\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d794f994-d939-48de-a909-cf2fc8b5c4b5\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"274cac8e-efc7-40a5-a972-2442fac09483\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"274cac8e-efc7-40a5-a972-2442fac09483\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f5343fe6-d867-4181-be4e-0591c5646161\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"f5343fe6-d867-4181-be4e-0591c5646161\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"52d7b2cd-7bbd-45ca-ac9a-b5906fc20171\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"52d7b2cd-7bbd-45ca-ac9a-b5906fc20171\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"67d153c5-c087-488b-ab56-a3bd408faf32\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"67d153c5-c087-488b-ab56-a3bd408faf32\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"85691ee0-e3e1-4b47-8749-f4abc7174d73\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"85691ee0-e3e1-4b47-8749-f4abc7174d73\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"6462e002-95e6-48da-a54f-9201f4796c27\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"6462e002-95e6-48da-a54f-9201f4796c27\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"388e8cd5-faf4-420a-b095-204bc245aefc\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"388e8cd5-faf4-420a-b095-204bc245aefc\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fb224e71-b578-45fc-a1b3-88101b1d1690\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"fb224e71-b578-45fc-a1b3-88101b1d1690\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"48af9987-16d0-4aa6-94de-13d847b4053c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"48af9987-16d0-4aa6-94de-13d847b4053c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 8900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"bf177fab-ea29-4327-8a49-382b23d48934\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"bf177fab-ea29-4327-8a49-382b23d48934\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9000
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5fff2e5a-0a88-4b05-bb81-4f535093357c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5fff2e5a-0a88-4b05-bb81-4f535093357c\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9100
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"4c2f812c-8bbc-439b-980b-a9b3c9fd280d\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"4c2f812c-8bbc-439b-980b-a9b3c9fd280d\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9200
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d91789fc-b06e-44fb-8f4a-5389fa7719b3\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d91789fc-b06e-44fb-8f4a-5389fa7719b3\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9300
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"a1638255-9dc7-43c6-8b62-f7741ac19780\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"a1638255-9dc7-43c6-8b62-f7741ac19780\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9400
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d60a5e68-6551-49a5-991e-1b620ffca2c5\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"d60a5e68-6551-49a5-991e-1b620ffca2c5\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9500
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"e6b054cc-7bc1-47c5-affa-b75fd566682e\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"e6b054cc-7bc1-47c5-affa-b75fd566682e\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9600
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"713cf00d-f219-44b3-ae8c-bc22656b02c1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"713cf00d-f219-44b3-ae8c-bc22656b02c1\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9700
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5531fcb6-23ea-4863-8868-5dd4807ef254\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"5531fcb6-23ea-4863-8868-5dd4807ef254\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9800
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"247b0b09-a613-4c0b-b462-9e57d8ab380d\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"247b0b09-a613-4c0b-b462-9e57d8ab380d\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 9900
}
{
  "sandboxLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"11474d7e-4fde-4d79-8dcc-cc92eb861462\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "lambdaLog": "[ERROR] AttributeError: 'Trader' object has no attribute 'generate_price'\\n{\"errorMessage\": \"'Trader' object has no attribute 'generate_price'\", \"errorType\": \"AttributeError\", \"requestId\": \"11474d7e-4fde-4d79-8dcc-cc92eb861462\", \"stackTrace\": [\"  File \\\"/var/task/app.py\\\", line 31, in lambda_handler\\n    orders, conversion, traderData = trader.run(trading_state)\\n\", \"  File \\\"/var/task/trader.py\\\", line 80, in run\\n    acceptable_price = self.generate_price(\\n\"]}",
  "timestamp": 10000
}
{
  "sandboxLog": "Lambda errors have reached maximum value for submission f4a64905-606a-478e-8ede-8df106f4fd2b",
  "lambdaLog": null,
  "timestamp": 165507
}



Activities log:
day;timestamp;product;bid_price_1;bid_volume_1;bid_price_2;bid_volume_2;bid_price_3;bid_volume_3;ask_price_1;ask_volume_1;ask_price_2;ask_volume_2;ask_price_3;ask_volume_3;mid_price;profit_and_loss




Trade History:
null